tests/integration/test_integration.py

```diff
@@ -33,6 +33,25 @@ async def _run(selenium):
     _run(selenium_standalone_micropip)
 
 
+@integration_test_only
+def test_integration_install_no_deps(selenium_standalone_micropip, pytestconfig):
+    @run_in_pyodide
+    async def _run(selenium):
+        import micropip
+
+        await micropip.install("pyodide-micropip-test", deps=False)
+
+        try:
+            # pyodide-micropip-test depends on snowballstemmer
+            import snowballstemmer  # noqa: F401
+        except ModuleNotFoundError:
+            pass
+        else:
+            raise Exception("Should raise!")
```

Comment:
As the intention here is that we want `import snowballstemmer` to not be installed due to `deps=False`, is there a better way to handle this?

i.e., is it possible to wrap it in a context manager with `with pytest.raises(ModuleNotFoundError` (or we have to do it in the current way because `pytest` isn't installed?)?

Reply:
> i.e., is it possible to wrap it in a context manager with with pytest.raises(ModuleNotFoundError (or we have to do it in the current way because pytest isn't installed?)?

Yeah, good point. `pytest.raises` would work too, but I wanted to keep the test simple, not loading any extra package.